ENH:toto-run:handle multiple products and materials

SantiagoTorres opened the pull request:
This fixes the argparse calls so that the --materials and --products
flags can recieve multiple paths as arguments without the need of quotes
or commas.

SantiagoTorres committed:
FIX: toto-run: handle multi-path products and materials

This fixes the argparse calls so that the --materials and --products
flags can recieve multiple paths as arguments without the need of quotes
or commas.

lukpueh commented on toto/toto-run.py:
```diff
@@ -93,8 +93,8 @@ def main():
   parser = argparse.ArgumentParser(
       description="Executes link command and records metadata",
       usage="toto-verify.py --step-name <unique step name>\n" +
```
Although it's not part of this PR. This should be `toto-run.py` and not `toto-verify.py`. Do you want to fix this here, or do we need an extra PR for this, or should we just hot push to develop?

lukpueh commented on toto/toto-run.py:
```diff
@@ -132,9 +132,9 @@ def main():
   if args.products:
-    product_list = args.products.split(",")
+    product_list = args.products
 
```
Any reason not to add `args.materials` and `args.products` directly to the function call in [`in_toto_run`](https://github.com/in-toto/in-toto/pull/32/files#diff-8e16d631a0ef00bfa92de5d2ad6c1f65R139)?

SantiagoTorres replied:
Not at all, I wanted to not pollute the diff (so that if we need to add any validation to it we donn't have problems). I could do so if you think this is better.

lukpueh replied:
Hm. I'm fine with you polluting the diff.

lukpueh commented on toto/toto-run.py:
```diff
@@ -108,9 +108,9 @@ def main():
 
   # XXX LP: We should allow path wildcards here and sanitze them
```
I guess you can delete this comment now.

SantiagoTorres replied:
Will do.

SantiagoTorres committed:
FIX:toto-run: fixes program name in help text

The help text used to say "toto-verify" instead of "toto-run"

SantiagoTorres committed:
FIX:toto-run: addresses Lukas's comments

Here I:

- Removed the outdated comment on wildcard handling

- Removed the products_list and materials_list variables as they are not
  required anymore

SantiagoTorres commented:
@lukpueh Ready for another round 👍
